.github/workflows/ci.yml

```diff
@@ -131,6 +131,13 @@ jobs:
           cp LICENSE ./mithril-client-wasm/
           wasm-pack pack mithril-client-wasm/pkg
 
+      - name: Start a fake aggregator
+        shell: bash
+        run: |
+          cargo run -p mithril-aggregator-fake -- -p 8000 &
+          # Wait for the server to be ready
+          while ! nc -z localhost 8000; do sleep 1; done
```

Comment:
I've added a 60 seconds timeout.
I can't test it right now because of a conflict with the branch [greg/1397/fake_aggregator ](https://github.com/input-output-hk/mithril/tree/greg/1397/fake_aggregator) that prevents the CI from starting.
I'll test it by adding a sleep in the fake aggregator once the PR is merged.